tools/droplets/fabric/fabfile.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+import fabric
+from fabric.api import run, put, puts, task, env
+import os
+
+env.user = "root"
+
+def clean_up() -> None:
+    """
+    Clean up remote machine before taking snapshot.
+    """
+    run("rm -rf /tmp/* /var/tmp/*")
+    run("history -c")
+    run("cat /dev/null > /root/.bash_history")
+    run("unset HISTFILE")
+    run("apt-get -y autoremove")
+    run("apt-get -y autoclean")
+    run(br"find /var/log -mtime -1 -type f -exec truncate -s 0 {} \;")
```

Comment:
This one for example used to be `run("find /var/log -mtime -1 -type f -exec truncate -s 0 {} \;")`. pycodestyle was giving this error
```
 tools/droplets/fabric/fabfile.py:19:65: W605 invalid escape sequence '\;'
```